php/libraries/NDB_BVL_Feedback.class.inc

```diff
@@ -194,17 +194,19 @@ class NDB_BVL_Feedback
         $query .= " LIMIT 1";
 
         $result = $db->pselect($query, $qparams);
-
         if (!is_array($result) || count($result) == 0) {
             throw new Exception(
                 "Error, unable to select data for the feedback object"
             );
         }
 
         $this->_feedbackCandidateProfileInfo['CandID'] = $result[0]['CandID'];
-        if (isset($result[0]['SessionID'])) {
+        if (isset($result[0]['SessionID'])
+            || isset($this->_feedbackObjectInfo['SessionID'])
+        ) {
             $this->_feedbackCandidateProfileInfo['SessionID']
-                = $result[0]['SessionID'];
+                = $result[0]['SessionID'] ? $result[0]['SessionID']
```

Comment:
This seems like a perfect place to use the PHP7 null coalesce operator (`??`) to make this cleaner.